tests/packages/limited-api/meson.build

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2023 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+project('limoted-api', 'c', version: '1.0.0')
+
+py = import('python').find_installation()
```

Comment:
minor: probably good to add `pure: false` here. There's no `py.install_sources` so it doesn't technically matter here, but it would if a `.py` file was added.